File: atividade-4/019_unico/019_unico.hs
module Main where

-- Checar se n é unico em um vetor xs

unico :: Int -> [Int] -> Bool
unico n xs = (length [x | x <- xs, x == n]) == 1

main = do
  a <- readLn :: IO Int
  b <- readLn :: IO [Int]
  print $ unico a b

File: atividade-4/019_unico/019_unico.js
//Checar se n é unico em um vetor xs

const list = [1, 1, 2, 6, 6, 6, 7, 7, 7];
const number = 2;

const unico = (n, xs) => xs.filter((x) => x === n).length === 1;

console.log(unico(number, list));
